scripting/scripting_golang.go

```diff
@@ -122,10 +122,10 @@ func (e *golangEnvironment) Cleanup(ctx context.Context) error {
 	switch mgr := e.manager.(type) {
 	case remote:
 		return errors.Wrapf(mgr.CreateCommand(ctx).SetOutputOptions(e.opts.Output).AppendArgs("rm", "-rf", e.opts.Gopath).Run(ctx),
-			"problem removing remote golang environment '%s'", e.opts.Gopath)
+			"removing remote golang environment '%s'", e.opts.Gopath)
```

Comment:
Should `golang` be `Golang` or `Go`?

Reply:
Changed to `Golang`. I prefer not to use `Go` since it's difficult to search for.